<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FetchDialog</class>
 <widget class="QDialog" name="FetchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>223</width>
    <height>227</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>223</width>
    <height>227</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выборка</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images_ui/images/fetch.png</normaloff>:/images_ui/images/fetch.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_source">
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Источник:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="source"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_begin">
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Начало:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateStart">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_end">
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Конец:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEnd">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_style">
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Жанр:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="style"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_period">
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Период:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="period"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_filter">
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Фильтр:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filter"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FetchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FetchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
